app/src/main/java/org/wikipedia/descriptions/DescriptionEditView.java

```diff
@@ -371,6 +373,18 @@ void validateText() {
         updateSaveButtonEnabled();
     }
 
+    private void checkKeyboardLanguage(String text) {
```

Comment:
This no longer has to do with the "keyboard" language, so this can be named more intuitively, e.g. `detectLanguageFromText()`